src/facade/dragonfly_connection.cc

```diff
@@ -288,6 +297,19 @@ void Connection::UnregisterShutdownHook(ShutdownHandle id) {
   }
 }
 
+bool Connection::IsReplicaCommand(util::tls::TlsSocket::Buffer buff) const {
```

Comment:
So this PR is not relevant anymore, right?

Reply:
some of it. The tests and notls flag on replica is useful -- I will push the clean up today